indico/modules/events/editing/controllers/frontend.py

```diff
@@ -25,6 +25,14 @@ def _process_args(self):
         if not self.editable:
             raise NotFound
 
+    def _check_access(self):
+        RHContributionEditableBase._check_access(self)
+
+        if self.event.can_manage(session.user) or self.event.can_manage(session.user, permission='paper_editing'):
```

Comment:
the second check already includes full management permissions; no need to do two checks here